examples/charts/horizon.py

```diff
@@ -25,7 +25,8 @@
 output_file("horizon.html")
 
 hp = Horizon(
-    xyvalues, index='Date',
+    xyvalues, x_names=['Date'],
```

Comment:
ping @pzwang about this naming proposal and if we should support also `index` on some charts (so users can explicitly pass an iterable to be used as `index`). If so, users should specify `x_names` or `index` but not both.